src/upload.js

```diff
@@ -124,7 +124,7 @@ SkynetClient.prototype.uploadDirectory = async function (path, customOptions = {
   }
 
   const formData = new FormData();
-  path = p.normalize(path);
+  path = p.resolve(path);
```

Comment:
nit: Requiring `path` as `p` is not helpful to the readability of the code.

Reply:
https://github.com/SkynetLabs/nodejs-skynet/issues/53